config.ru

```diff
@@ -4,5 +4,6 @@ require ::File.expand_path('../config/environment',  __FILE__)
 # apply a prefix to the application, if one is defined
 # e.g. http://some.server.com/prefix where '/prefix' is defined by env variable
 map ENV['RAILS_RELATIVE_URL_ROOT'] || '/'  do
+  use Rack::Deflater
```

Comment:
Looks like I didn't pay enough attention and just ran 'passenger start' w/o RAILS_ENV=production, I'll reopen if I can figure this out better but at first glance I cannot reproduce these benchmarks on production :/